projects/kit/components/accordion/accordion-item/accordion-item.component.ts

```diff
@@ -88,8 +89,11 @@ export class TuiAccordionItemComponent
     @Output()
     readonly openChange = new EventEmitter<boolean>();
 
+    @ContentChild(TuiAccordionItemEagerContentDirective)
+    readonly eagerContent?: TuiAccordionItemEagerContentDirective;
+
     @ContentChild(TuiAccordionItemContentDirective)
-    readonly content?: TuiAccordionItemContentDirective;
+    readonly lazyContent?: TuiAccordionItemContentDirective;
```

Comment:
breaking changes?

Reply:
No, this is internal, it's only public because it is used in the template.